src/utils/vedleggUtils.test.ts

```diff
@@ -122,6 +124,45 @@ describe("VedleggUtilsTest", () => {
         expect(actualMetadata).toBe(expectedEttersendelseMetadata);
     });
 
+    it("should log error for empty file file object ettersendelse", () => {
+        const formDataMedTomFil: FormData = opprettFormDataMedVedleggFraFiler([pngFile, jpgFile, emptyFile]);
+        expect(formDataMedTomFil).toBeDefined();
+
+        const formDataMedTomFilEntryValues: FormDataEntryValue[] = formDataMedTomFil.getAll("files");
+        expect(formDataMedTomFilEntryValues).toBeDefined();
+        expect(formDataMedTomFilEntryValues.length).toBe(3);
+
+        [
+            {file: formDataMedTomFilEntryValues[0], filnavn: "metadata.json"},
+            {file: formDataMedTomFilEntryValues[1], filnavn: pngFile.filnavn},
+            {file: formDataMedTomFilEntryValues[2], filnavn: jpgFile.filnavn},
+        ].forEach((value: {file: FormDataEntryValue; filnavn: string}) => {
+            const file = value.file as File;
+            expect(file).toBeDefined();
+            expect(file.name).toBe(value.filnavn);
+        });
+    });
+
+    it("should find fileetternavn", () => {
```

Comment:
Vurder å rename testtittler også.

Reply:
Jeg gjør dette også, for jeg er så medgjørlig idag ;-)

Reply:
Done